app/src/main/java/org/thoughtcrime/securesms/onboarding/LandingActivity.kt

```diff
@@ -19,28 +85,174 @@ class LandingActivity : BaseActionBarActivity() {
+                listState.animateScrollToItem(0.coerceAtLeast((count - 1)))
+                delay(1500L)
+            }
+        }
+
+        Column {
+            Column(modifier = Modifier
+                .weight(1f)
+                .padding(horizontal = LocalDimensions.current.marginMedium)) {
+                Spacer(modifier = Modifier.weight(1f))
+                Text(
+                    stringResource(R.string.onboardingBubblePrivacyInYourPocket),
+                    modifier = Modifier.align(Alignment.CenterHorizontally),
+                    style = MaterialTheme.typography.h4,
+                    textAlign = TextAlign.Center
+                )
+                Spacer(modifier = Modifier.height(LocalDimensions.current.itemSpacingMedium))
+
+                LazyColumn(
+                    state = listState,
+                    modifier = Modifier
+                        .heightIn(min = 200.dp)
+                        .fillMaxWidth()
+                        .weight(3f),
+                    verticalArrangement = Arrangement.spacedBy(LocalDimensions.current.itemSpacingSmall)
+                ) {
+                    items(
+                        MESSAGES.take(count),
+                        key = { it.stringId }
+                    ) { item ->
+                        AnimateMessageText(
+                            stringResource(item.stringId),
+                            item.isOutgoing
+                        )
+                    }
+                }
+
+                Spacer(modifier = Modifier.weight(1f))
+            }
+
+            Column(modifier = Modifier.padding(horizontal = LocalDimensions.current.marginLarge)) {
+                FilledButton(
+                    text = stringResource(R.string.onboardingAccountCreate),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.CenterHorizontally)
+                        .contentDescription(R.string.AccessibilityId_create_account_button),
+                    onClick = ::startPickDisplayNameActivity
+                )
+                Spacer(modifier = Modifier.height(LocalDimensions.current.itemSpacingSmall))
+                OutlineButton(
+                    text = stringResource(R.string.onboardingAccountExists),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.CenterHorizontally)
+                        .contentDescription(R.string.AccessibilityId_restore_account_button)
+                ) { start<LinkDeviceActivity>() }
+                BorderlessButton(
+                    text = stringResource(R.string.onboardingTosPrivacy),
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.CenterHorizontally)
+                        .contentDescription(R.string.AccessibilityId_open_url),
+                    onClick = ::openDialog
```

Comment:
This dialog is missing an 'x' close button

Reply:
@bemusementpark

Reply:
done